[DOC-12715] Document New Search Request Params for Search Fine-Tuning (#291)
couchbaselabs/docs-devex: modules/search/pages/search-request-params.adoc

|====

[#knn-params]
=== knn params Object 

Use the `params` object inside a `knn` object to fine tune the probes and centroids the Search Services uses and searches while running a Vector Search request. 

The `params` object can contain the following properties: 

[cols="1,1,1,4"]
|====
|Property |Type |Required? |Description

|ivf_nprobe_pct |Number (percentage) |No a|

> Set the `ivf_nprobe_pct` value to control the percentage of clusters that the Search Service searches during a single Vector Search query, or the percentage of probes used.

Reviewer: ```suggestion
Set the `ivf_nprobe_pct` value to control the percentage of probes (i.e. the percentage of clusters) that the Search Service searches during a single Vector Search query.
```

Reviewer: Small suggestion here 👆🏾
